worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/blocks/CharBlocks.java

```diff
@@ -204,7 +204,7 @@ public char get(int x, int y, int z) {
 
     // Not synchronized as it refers to a synchronized method and includes nothing that requires synchronization
     public void set(int x, int y, int z, char value) {
-        final int layer = y >> 4;
+        final int layer = (y >> 4) - minSectionPosition;
```

Comment:
This should not be changed. The set method called afterwards applies the normalisation

Reply:
Done